packages/amplify-category-geo/src/service-utils/geofenceCollectionUtils.ts

```diff
@@ -2,7 +2,7 @@ import {
   $TSContext, $TSObject, stateManager, pathManager,
 } from 'amplify-cli-core';
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-import { App } from '@aws-cdk/core';
+import { App } from 'aws-cdk-lib';
```

Comment:
here we import { App } from 'aws-cdk-lib' but in other files, we are importing * from 'aws-cdk-lib' and using *.app for example. Can we make this consistent?

Reply:
I guess both are fine. I am not including this as this will increase the count of files for this PR I think.

Reply:
@evcodes . Please keep in mind that we're facing a period of merges between dev and this branch.
Any non-functional changes should be parked for the sake of easing that process.